x-pack/plugins/cases/public/containers/constants.ts

```diff
@@ -25,6 +25,7 @@ export const casesQueriesKeys = {
   case: (id: string) => [...casesQueriesKeys.caseView(), id] as const,
   caseFiles: (id: string, params: unknown) =>
     [...casesQueriesKeys.case(id), 'attachments', params] as const,
+  caseFileStats: (id: string) => [...casesQueriesKeys.case(id), 'stats'] as const,
```

Comment:
Maybe it is time to make the files query keys more granular. For example

```
caseFiles: (id: string) => [...casesQueriesKeys.case(id), 'files'] as const,
  caseFilesTab: (id: string, params: unknown) =>
    [...casesQueriesKeys.case(id), 'files', params] as const,
  caseFileStats: (id: string) => [...casesQueriesKeys.caseFiles(id), 'stats'] as const,
```

Reply:
If we add `caseFilesTable` who would actually use `caseFiles`? There are only these two calls.

Reply:
It can be used to invalidate all queries only for the files. For example `queryClient.invalidateQueries(casesQueriesKeys.caseFiles)`. But I made a mistake, it should be `caseFiles: () => [...casesQueriesKeys.caseView(), 'files'] as const` (no case id in the key).

Reply:
> it should be `caseFiles: () => [...casesQueriesKeys.caseView(), 'files'] as const` 

I am pretty sure this is what I had initially in the previous PR and you suggested changing it to `casesQueriesKeys.case(id)` 😆

Reply:
🙂 Yes, but what you had thought was for the file list which should have the case ID. What I suggested here is a bit different. You have a category for all file queries AND two more for each query which are bound to the case ID. It is not the same.

Reply:
How about

```
case: (id: string) => [...casesQueriesKeys.caseView(), id] as const,
caseFiles: (id: string, params: unknown) =>
  [...casesQueriesKeys.case(id), 'files', params] as const,
caseFileStats: (id: string) => [...casesQueriesKeys.case(id), 'files', 'stats'] as const,
```
?